msteams-platform/tabs/what-are-custom-tabs.md

```diff
@@ -6,40 +6,48 @@ ms.topic: overview
+There are two types of tabs available in Teams - channel/group and personal. A channel/group tab delivers content to channels and group chats, and are a great way to create collaborative spaces around dedicated web-based content. Personal tabs, along with direct conversation bots, are part of personal apps and are scoped to a single user. They can be pinned to the left navigation bar and promote increased productivity by making your service available directly inside the Teams client.
 
 ## Tabs user scenarios
 
-**Scenario:** Make an existing web-based resource available inside Microsoft Teams.
-**Example:** You have an informational corporate website you make available as a personal tab in Teams.
+**Scenario:** Bring an existing web-based resource inside Teams. \
+**Example:** You create a personal tab in your Teams app that presents an informational corporate website to users.
 
-**Scenario:** You want to add an about or help page for your bot or messaging extension.
-**Example:** You have a static webpage that you make available as a personal tab in Teams.
+**Scenario:** Add static support pages to a Teams bot or messaging extension. \
+**Example:** You create personal tabs that provide about and help webpage content to users.
 
-**Scenario:** You have items in a system that your users need to discuss or collaborate around regularly.
-**Example:** You create a group/channel tab that is configurable with deep linking to individual items.
+**Scenario:** Provide accessibility to system-level items your users access regularly for cooperative dialogue and collaboration. \
+**Example:** You create a configurable channel/group tab with deep linking to individual items.
 
-## How do tabs work?
+## How do tabs function?
 
-All Teams development (custom tabs, messaging extensions, or bots) needs to be bundled in a [Teams app package](foo.md) for distribution either in the Teams App Store (AppSource), to your organization or within a team. A custom tab is declared directly in the manifest of your app package. Your app can have a maximum of one (1) group/channel tab and up to sixteen (16) personal tabs
+Teams development and integration (custom tabs, messaging extensions, or bots) need to be bundled in a [Teams app package](foo.md) for distribution either in the Teams App Store ([AppSource](https://appsource.microsoft.com)), to your organization, or within a team. A custom tab is declared directly in the manifest of your app package. Your app can have a maximum of one (1) channel/group tab and up to sixteen (16) personal tabs
```

Comment:
The first sentence in this section is rough, consider rewording. Maybe something like "All Teams customizations need to be bundled.."? Integrations don't necessarily need to be bundled in an app manifest, as you can define a webhook w/o an app manifest.